boards/stm32f4discovery/include/periph_conf.h

```diff
@@ -29,6 +29,32 @@
 extern "C" {
 #endif
 
+/**
+ * @name    DMA streams configuration
+ * @{
+ */
+#ifdef MODULE_PERIPH_DMA
+static const dma_conf_t dma_config[] = {
+    { .stream = 11 },   /* DMA2 Stream 3 - SPI1_TX */
+    { .stream = 10 },   /* DMA2 Stream 2 - SPI1_RX */
+    { .stream = 4 },    /* DMA1 Stream 4 - SPI2_TX */
+    { .stream = 3 },    /* DMA1 Stream 3 - SPI2_RX */
+    { .stream = 5 },    /* DMA1 Stream 5 - SPI3_TX */
```

Comment:
SPI3 is not used on this board. Not sure we want to keep it ?

Reply:
I removed it, let's keep this clean